source/macos/AdaptiveCards/AdaptiveCardsTests/Renderers/ContainerRendererTests.swift

```diff
@@ -40,13 +41,14 @@ class ContainerRendererTests: XCTestCase {
         
         container = .make(verticalContentAlignment: .bottom, items: [FakeTextBlock.make()])
         containerView = renderContainerView(container)
+        // since we removed padding view all together and replaced with lastPadding but use paddingView for vertical content alignment this has changed
         // SpaceView 1
-        XCTAssertEqual(containerView.stackView.arrangedSubviews.capacity, 2)
+        XCTAssertEqual(containerView.stackView.arrangedSubviews.capacity, 3)
```

Comment:
Please add comment for stretchable views is hide when there is other stretchable views are present.